script/Makefile

```diff
@@ -402,7 +402,11 @@ maven-overlay:
 kamel-overlay:
 	@echo "####### Copying kamel CLI to output build directory..."
 	mkdir -p build/_output/bin
-	cp kamel build/_output/bin
+  ifeq ($(shell uname -m), arm64)
```

Comment:
The problem appears for both macOS running on `x86_64` and on `arm64` - a macOS compiled library will never work inside a Linux container image.

why not 

`ifneq ($(shell uname -m), Linux)`

?
If it's anything else than Linux, put the linux binary inside the image

Reply:
The separate `kamel.linux.arm` binary is only built for arm64 based os especially for this architecture. so we need to distinguish on the different types of archs as there will be individual kamel linux binaries. 

I wonder because in my former macOS x64 laptop this has been working as expected. so I need to check for the history of the Makefile what change has been causing this.